test/Api.Test/SecretsManager/Controllers/SecretsControllerTests.cs

```diff
@@ -447,4 +519,30 @@ private static Guid SetOrganizations(ref List<Secret> data)
 
         return organizationId;
     }
+
+    private static DateTime? SetupSecretsSyncRequest(bool nullLastSyncedDate, List<Secret> secrets,
+        SutProvider<SecretsController> sutProvider, Guid organizationId)
+    {
+        var lastSyncedDate = GetLastSyncedDate(nullLastSyncedDate);
+
+        sutProvider.GetDependency<ICurrentContext>().AccessSecretsManager(Arg.Is(organizationId))
+            .ReturnsForAnyArgs(true);
+        sutProvider.GetDependency<IAccessClientQuery>()
+            .GetAccessClientAsync(Arg.Any<ClaimsPrincipal>(), Arg.Any<Guid>())
+            .Returns((AccessClientType.ServiceAccount, new Guid()));
+        sutProvider.GetDependency<ISecretsSyncQuery>().GetAsync(Arg.Any<SecretsSyncRequest>())
+            .Returns((true, secrets));
+        return lastSyncedDate;
+    }
+
+    private static DateTime? GetLastSyncedDate(bool nullLastSyncedDate)
+    {
+        DateTime? lastSyncedDate = DateTime.UtcNow.AddDays(-1);
+        if (nullLastSyncedDate)
+        {
+            lastSyncedDate = null;
+        }
+
+        return lastSyncedDate;
```

Comment:
Already approved above, just adding a small optional item. I think this could potentially be one line?

```suggestion
        return nullLastSyncedDate ? null : DateTime.UtcNow.AddDays(-1);
```

Reply:
Good suggestion updated in https://github.com/bitwarden/server/pull/3906/commits/d126f899eddc770606864654ce3a8a38ce541e83